README.md

```diff
@@ -129,6 +130,35 @@ Same button using DI:
 ))
 ```
 
+### Helping users report technical issues
+
+The [hmrcReportTechnicalIssueHelper](src/main/play-26/twirl/uk/gov/hmrc/hmrcfrontend/views/helpers/hmrcReportTechnicalIssueHelper.scala.html) component 
+generates a link that allows users to report technical issues with your service.
+
+To configure this helper, add the following configuration to your `application.conf`
+
+```
+contact-frontend.serviceCode = "<any-service-code>"
+```
+
+serviceCode helps identify your service when members of the public report technical issues. 
```

Comment:
I'm wondering if we should rename this to serviceId to be consistent with our other documentation on this: https://github.com/hmrc/play-frontend-hmrc/pull/31/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R144